recipients_app/clean_build.sh

```
@@ -0,0 +1,24 @@
+#!/bin/zsh
```

Shellcheck (0.10.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)